client/model/device/status/status.go

```diff
@@ -0,0 +1,14 @@
+package status
+
+type Type string
+
+// see also: https://github.com/cisco-lockhart/lh-plugin-core/blob/master/lh-target/src/main/java/com/cisco/lockhart/target/data/ServiceActivationState.java
```

Comment:
Should I remove this link to internal repo? I feel like keeping it would be beneficial to future developer, as they are likely to be a person from CDO rather than external developer.